atlasdb-tests-shared/src/test/java/com/palantir/atlasdb/schema/indexing/generated/DataTable.java

```diff
@@ -12,16 +10,11 @@
 import java.util.SortedMap;
 import java.util.concurrent.Callable;
 import java.util.concurrent.ExecutorService;
-import java.util.concurrent.TimeUnit;
```

Comment:
I suspect we should find a way to deal with the warnings or to fix the code-gen, but for now I'd skip checking in the changes in the generated classes in to prevent future churn.